### PR TITLE
docs: move lazy loading task from router to lazy loading doc

### PR DESCRIPTION
**Pull request by kapunahelewong**
## PR Checklist
Please check if your PR fulfills the following requirements:

- [ ] The commit message follows our guidelines: https://github.com/angular/angular/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type
What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [x] Documentation content changes
- [ ] angular.io application / infrastructure changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?

Moves lazy loading task to lazy loading doc. A task is a quick reference. Changes headings levels (and some header wording for context) for the step-by-step process.

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

**Review by atscott — COMMENTED** [review]
Is there still going to be a link to this Lazy-loading feature modules guide from the main router doc? It looks like it was removed in this change as well.

File: aio/content/guide/lazy-loading-ngmodules.md [hunk 1]
```diff
@@ -1,19 +1,52 @@
...
 
 For the final sample app with two lazy-loaded modules that this page describes, see the
 <live-example></live-example>.
 
+{@a lazy-loading}
+
+## High level view
+
+To lazy load Angular modules, use `loadchildren` (instead of `component`) in your `AppRoutingModule` `routes` configuration as follows:
+
```
Comment (jbogarthyde) [inline aio/content/guide/lazy-loading-ngmodules.md:14]: minor nit -- style guide says to end intros with period, not colon.

File: aio/content/guide/lazy-loading-ngmodules.md [hunk 2]
```diff
@@ -1,19 +1,52 @@
...
+  {
+    path: 'items',
+    loadChildren: () => import('./items/items.module').then(m => m.ItemsModule)
+  }
+];
+
+</code-example>
+
+In the lazy loaded module's routing module, add a route for the component.
+
```
Comment (jbogarthyde) [inline aio/content/guide/lazy-loading-ngmodules.md:27]: lazy-loaded

File: aio/content/guide/lazy-loading-ngmodules.md [hunk 3]
```diff
@@ -1,19 +1,52 @@
 # Lazy-loading feature modules
 
-## High level view
-
 By default, NgModules are eagerly loaded, which means that as soon as the app loads, so do all the NgModules, whether or not they are immediately necessary. For large apps with lots of routes, consider lazy loading&mdash;a design pattern that loads NgModules as needed. Lazy loading helps keep initial
 bundle sizes smaller, which in turn helps decrease load times.
 
 For the final sample app with two lazy-loaded modules that this page describes, see the
 <live-example></live-example>.
```
Comment (jbogarthyde) [inline aio/content/guide/lazy-loading-ngmodules.md:8]: We've been putting these in alert divs, haven't we?

File: aio/content/guide/lazy-loading-ngmodules.md [hunk 4]
```diff
@@ -1,19 +1,52 @@
...
 
+{@a lazy-loading}
+
+## High level view
+
```
Comment (jbogarthyde) [inline aio/content/guide/lazy-loading-ngmodules.md:12]: Maybe something like "Lazy loading basics", then an intro with something like 

This section introduces the basic procedure for converting a route to be lazy-loaded.
For a step-by-step example, see (link to anchor) on this page.

**Review by jbogarthyde — COMMENTED** [review]
Some suggestions

**Comment by kapunahelewong** [comment]
Thanks for pointing that out, @atscott. You're so right! Added a small section with a link to the doc.

File: aio/content/guide/router.md [hunk 5]
```diff
@@ -357,42 +357,18 @@ Inject `ActivatedRoute` and `Router` in the constructor of the component class s
 
 {@a lazy-loading}
```
Comment (aikidave) [inline aio/content/guide/router.md:358]: Does this line need to be deleted?

File: aio/content/guide/lazy-loading-ngmodules.md [hunk 6]
```diff
@@ -1,19 +1,61 @@
 # Lazy-loading feature modules
```
Comment (aikidave) [inline aio/content/guide/lazy-loading-ngmodules.md:1]: This topic could really use a refresh. There's a [Jira work item](https://angular-team.atlassian.net/browse/DOCS-741?atlOrigin=eyJpIjoiNTIwZDc1MDg1YWE0NDgzMjhmODcxMzBjN2I2ZmIzMTMiLCJwIjoiaiJ9) for that work. If we are moving content from the router.md to this one, maybe we should prioritize updating this topic?

**Review by aikithoughts — CHANGES_REQUESTED** [review]

File: aio/content/guide/lazy-loading-ngmodules.md (hunk 6 above)
Comment (kapunahelewong) [reply aio/content/guide/lazy-loading-ngmodules.md:1]: Would you like that rewrite to be part of this PR, @aikidave?

File: aio/content/guide/router.md (hunk 5 above)
Comment (kapunahelewong) [reply aio/content/guide/router.md:358]: There is still a section here for lazy loading that then links to the lazy loading page (per [this comment](https://github.com/angular/angular/pull/36748#pullrequestreview-397684699)). The next push ought to clarify.

File: aio/content/guide/lazy-loading-ngmodules.md (hunk 6 above)
Comment (kapunahelewong) [reply aio/content/guide/lazy-loading-ngmodules.md:1]: Per off-line conversation we will do this Jira in a second PR.

File: aio/content/examples/binding-syntax/e2e/src/app.e2e-spec.ts [hunk 7]
```diff
@@ -1,4 +1,4 @@
-import { browser, element, by } from 'protractor';
+  import { browser, element, by } from 'protractor';
```
Comment (AndrewKushnir) [inline aio/content/examples/binding-syntax/e2e/src/app.e2e-spec.ts:1]: Is this change required?

File: aio/content/guide/lazy-loading-ngmodules.md [hunk 8]
```diff
@@ -175,6 +215,101 @@ The `forRoot()` method takes care of the *global* injector configuration for the
...
+In your service, import the following router members, implement `Resolve`, and inject the `Router` service:
+
+<code-example header="Resolver service (excerpt)">
+
+import { Resolve } from '@angular/router';
+
+...
+
+export class CrisisDetailResolverService implements Resolve<> {
+  resolve(route: ActivatedRouteSnapshot, state: RouterStateSnapshot): Observable<> {
```
Comment (AndrewKushnir) [inline aio/content/guide/lazy-loading-ngmodules.md:270]: This example code looks a bit suspicious: we omit generic types for `Resolve` and `Observable`. This code will most likely cause compile-time errors if copied as is. I see that the same code already exists and it was just moved in this PR to a different place, so may be we can create a ticket and look into fix this example later?

**Review by AndrewKushnir — APPROVED** [review]
LGTM, just left a couple comments. Thanks @kapunahelewong 👍

File: aio/content/examples/binding-syntax/e2e/src/app.e2e-spec.ts (hunk 7 above)
Comment (kapunahelewong) [reply aio/content/examples/binding-syntax/e2e/src/app.e2e-spec.ts:1]: Nope! And unintentional. Fixed. Thanks!

File: aio/content/guide/lazy-loading-ngmodules.md (hunk 8 above)
Comment (kapunahelewong) [reply aio/content/guide/lazy-loading-ngmodules.md:270]: Thanks for catching that! Issue filed: https://github.com/angular/angular/issues/37113

File: aio/content/guide/lazy-loading-ngmodules.md [hunk 9]
```diff
@@ -175,6 +215,101 @@ The `forRoot()` method takes care of the *global* injector configuration for the
 The `forChild()` method has no injector configuration. It uses directives such as `RouterOutlet` and `RouterLink`.
 For more information, see the [`forRoot()` pattern](guide/singleton-services#forRoot) section of the [Singleton Services](guide/singleton-services) guide.
 
+{@a preloading}
+
+## Preloading
+
+Preloading improves UX by loading parts of your app in the background.
```
Comment (atscott) [inline aio/content/guide/lazy-loading-ngmodules.md:222]: I feel like this is a tad bit incomplete, but I can also understand not wanting to be too verbose. This seems to beg the question: "Why does loading parts of the app in the background improve UX?" The `preloading component data` section addresses this: "You can preload component data so that..."

File: aio/content/guide/lazy-loading-ngmodules.md [hunk 10]
```diff
@@ -175,6 +215,101 @@ The `forRoot()` method takes care of the *global* injector configuration for the
...
+  path: '/your-path',
+  component: YourComponent,
+  resolve: {
+    crisis: YourResolverService
+  }
+}
+</code-example>
+
+
+In the component, use an observable to get the data from the `ActivatedRoute`.
```
Comment (atscott) [inline aio/content/guide/lazy-loading-ngmodules.md:298]:
```suggestion
In the component, use an `Observable` to get the data from the `ActivatedRoute`.
```

**Review by atscott — APPROVED** [review]

File: aio/content/guide/lazy-loading-ngmodules.md [hunk 11]
```diff
@@ -134,7 +174,7 @@ The other feature module's routing module is configured similarly.
 
 <code-example path="lazy-loading-ngmodules/src/app/orders/orders-routing.module.ts" id="orders-routing.module.ts" region="orders-routing-module-detail" header="src/app/orders/orders-routing.module.ts (excerpt)"></code-example>
 
-## Confirm it’s working
+### Confirm it’s working
```
Comment (aikidave) [inline aio/content/guide/lazy-loading-ngmodules.md:177]: "It's" is ambiguous. I suggest "Verify lazy loading" or similar.

**Review by aikithoughts — APPROVED** [review]
One minor suggestion. Otherwise LGTM
